cli/azd/resources/scaffold/templates/ai-project.bicept

```diff
@@ -125,6 +136,7 @@ resource hub 'Microsoft.MachineLearningServices/workspaces@2024-10-01' = {
     }
     v1LegacyMode: false
     publicNetworkAccess: 'Enabled'
+    systemDatastoresAuthMode: 'Identity'
```

Comment:
By setting this, the Storage Blob Data Contributor and Storage File Data Privileged Contributor role assignments for the hub's system-assigned identity are automatically created during deployment:

<img width="481" alt="image" src="https://github.com/user-attachments/assets/4d526929-074a-4d44-a978-86b5e7f0d7b9" />